docker_images/ci-functional/Dockerfile

```diff
@@ -0,0 +1,84 @@
+FROM ubuntu:eoan
+
+LABEL maintainer="Conan.io <info@conan.io>"
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ENV PY27 /root/.pyenv/versions/2.7.17/bin/python
```

Comment:
keep all env vars on same docker layer, it helps keeping the docker image smaller.

    ENV PY35 /root/.pyenv/versions/3.5.9/bin/python \
        PY37 /root/.pyenv/versions/3.7.6/bin/python \
        ...